
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="icon" type="image/png" href="favicon.png"/>

    <title>Heath Lab – EEOB – Iowa State University</title>

    <!-- Bootstrap core CSS -->
    <link href="./css/bootstrap.min.css" rel="stylesheet">

    <!-- Custom styles for this template -->
    <link href="navbar-fixed-top.css" rel="stylesheet">

  </head>

  <body>

    <!-- fixed navbar -->
    <div class="navbar navbar-default navbar-fixed-top" role="navigation">
      <div class="container">
        <div class="navbar-header">
          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target=".navbar-collapse">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </button>
          <a class="navbar-brand" href="index.html">Heath Lab @ ISU</a>
        </div>
        <div class="navbar-collapse collapse">
          <ul class="nav navbar-nav">
            <li><a href="index.html">Home</a></li>
            <li class="active"><a href="research.html">Research</a></li>
            <li><a href="people.html">People</a></li>
            <li><a href="publications.html">Publications</a></li>
            <li><a href="software.html">Software</a></li>
            <li class="dropdown">
              <a href="#" class="dropdown-toggle" data-toggle="dropdown">Resources <span class="caret"></span></a>
              <ul class="dropdown-menu" role="menu">
                <li><a href="resources/links.html">Links</a></li>
                <li><a href="resources/collaborators.html">Collaborators</a></li>
                <li class="divider"></li>
                <li class="dropdown-header"><font color="white">Teaching</font></li>
                <li><a href="resources/workshops.html">Workshops</a></li>
                <li class="divider"></li>
                <li class="dropdown-header"><font color="white">Join the Heath Lab</font></li>
                <li><a href="resources/gradschool.html">Graduate Programs</a></li>
                <li><a href="resources/postdocinfo.html">Postdoc Opportunities</a></li>
              </ul>
            </li>
          </ul>
          <ul class="nav navbar-nav navbar-right">
            <li><a href="http://www.eeob.iastate.edu/">EEOB</a></li>
            <li><a href="http://www.iastate.edu/">ISU</a></li>
          </ul>
        </div><!--/.nav-collapse -->
      </div>
    </div>


    <div class="container">



	<div class="page-header">
        <h1 id="what">Research</h1>
    </div>
      <div class="jumbotron">
        <p>Our lab develops methods that broaden our understanding of the evolutionary processes responsible for generating the patterns we observe in the tree of life. Specifically, we develop novel computational and 
			statistical methods for estimating evolutionary parameters in a phylogenetic context. 
			We evaluate the properties and performance of phylogenetic methods using biological data sets and through the development of new simulation tools. Additionally, with collaborators, we apply phylogenetic techniques to empirical data sets to understand patterns of biodiversity, morphological evolution, and historical biogeography.</p>
      </div>


        <div class="row">
            <div class="col-lg-12">
                <h3 class="page-header">Combining Paleontological and Neontological Data for Phylogenetic Inference
                </h3>
			  <div class="well">
				<p>Historical observations in the form of fossil occurrence times or other geological data are 
				fundamental components necessary for inferring the absolute timing of speciation events. The 
				common approach to time calibration taken by most neontologists is to assign minimum age estimates, 
				based on fossil specimens, to nodes within their group of interest. However, we can rarely assign 
				fossils to nodes of a tree without error, and much of the information associated with fossil taxa is 
				lost when we try to represent the fossil record as a single time estimate applied to a single internal node.
				</p>
				<h4>The Fossilized Birth-Death Process</h4>
				<p>
				In collaboration with Tanja Stadler (ETH Zürich) and John Huelsenbeck (UC–Berkeley), I 
				developed a unified extant and extinct- species diversification model that eliminates the 
				need for the ad hoc prior densities that are common practice in Bayesian phylogenetics. The 
				‘fossilized birth-death’ (FBD) process is a model for calibrating divergence-time estimates 
				in a Bayesian framework and explicitly acknowledges that extant species and fossils are representatives 
				of the same macroevolutionary process. Under this model, we can estimate internal node ages conditional 
				on a tree of extant taxa and set of fossil occurrence times. We developed novel reversible-jump MCMC 
				methods to marginalize over realizations of this process and implemented them in a recent version of DPPDiv. 
				This model improves upon standard approaches for calibrating phylogenies with fossil data by allowing for 
				inclusion of all available fossils and providing coherent measures of statistical uncertainty. (See <a href="http://www.pnas.org/content/111/29/E2957.abstract">Heath et al., PNAS 2014</a>)
				</p>
<!-- 
				<h4>Fossil Phylogenetics with Trilobites as a Model System</h4>
				<p>
				For some taxonomic groups, the fossil record provides a wealth of information about the evolution of 
				morphological characters and lineage diversification. Because of this, the development of methods for 
				fully integrating information from fossil specimens into phylogenetic analyses is a critical area of 
				research. Toward this end, I am collaborating with Drs. Mark Holder and Bruce Lieberman (U. Kansas) 
				to develop improved methods for likelihood-based phylogenetic analysis of morphological data with an 
				emphasis on fossil taxa. As part of this project, I will integrate complex models of morphological 
				evolution and stratigraphic information into Bayesian methods for phylogenetic inference. 
				This work will build on the FBD model and estimate the explicit phylogenetic relationships of fossil 
				organisms. Furthermore, we will extend the diversification models to account for our current 
				understanding of the rock record and stratigraphy. We will harness the rich fossil record of the 
				Trilobita, specifically the well-studied groups Olenellina and Cheiruridae, to evaluate and test 
				new phylogenetic methods. Ultimately, this work will expand our understanding of the patterns of 
				diversification and morphological evolution in this group. Furthermore, our methods are naturally 
				applicable to data sets combining both modern and fossil species, thus contributing to current 
				research in molecular phylogenetics and macroevolution. This project is funded by NSF grant 
				<a href="http://phylo.bio.ku.edu/fossil/">DEB-1256993</a>.
				</p>
 -->
			  </div>
           </div>
        </div>

        <div class="row">
            <div class="col-lg-12">
                <h3 class="page-header">Co-Evolution in Figs and Fig Wasps
                </h3>
			  <div class="well">
				<p>In collboration with Dr. <a href="http://www.eeob.iastate.edu/faculty/nason/">John Nason</a> 
				(Iowa State University), Dr. <a href="http://www.stri.si.edu/english/scientific_staff/staff_scientist/scientist.php?id=15">E. Allen Herre</a> (Smithsonian Tropical Research Institute, Panama), 
				Dr. <a href="https://scholar.google.com/citations?user=fBVG0lkAAAAJ&hl=en">Charlotte Jandér</a> (Harvard University), 
				Dr. <a href="http://www.clfs.umd.edu/biology/machadolab/">Carlos Machado</a> (University of Maryland), and 
				Dr. <a href="http://www.nbb.cornell.edu/neurobio/ragusolab/index.html">Robert Raguso</a> (Cornell University),
				we are investigating the coevolutionary history of species interactions in 
				Central American figs and their pollinating (mutualistic) and non-pollinating (antagonistic) 
				fig wasps. 
				Figs and their fig wasp pollinators and parasites have co-evolved for ~90 million years to become both highly diverse (>750 species of figs) and ecologically important “keystone” components of tropical forest ecosystems. Figs and wasps have long been assumed to represent a case of strict co-speciation, with highly specific pollinator and parasitic (non-pollinator) wasps identifying appropriate hosts via distinctive volatile chemical signals. More recent studies suggest a more complex scenario, however, involving an evolutionary history punctuated by host-shifts by individual wasp species. Although the wasp associations with fig hosts have been widely studied, the genetic consequences for the host figs of host-shifting pollinators and the mechanisms underlying host recognition remain poorly understood. 
				</p>
				<p>
				This project will fill these gaps by producing robust, detailed, many-gene phylogenies for 14 strangling fig (<i>Ficus</i>) species and their associated pollinating (<i>Pegoscapus</i>) and non-pollinating (<i>Idarnes</i>) fig wasps (~60 species) from the vicinity of Barro Colorado Island, Panama. Using transcriptome sequences, we will target ~300 genes from each of three species per lineage for capture and subsequent Illumina sequencing. Phylogenies will be inferred using Bayesian methods and will enable robust testing of phylogenetic congruence between figs and fig wasps. Further, they will guide population-level genotype by sequencing to test a priori predictions of potential cases of hybridization in the figs and host shifting and race formation in both pollinator and non-pollinator wasps. Combined with quantification of wasp-attracting fig volatiles and fruit-surface chemicals, this work will detect and resolve the genomic consequences of host introgression due to host-shifting pollinator wasps, and link them to the chemical basis of host-recognition.
				</p>
				<p>
				This research will significantly clarify both the patterns and processes underlying the evolutionary ecology of fig and fig wasp interactions. Our standardized, genomic approach is essential for: 1) obtaining robust fig and fig wasp species trees, 2) delimiting fig species and discriminating cases of introgressive gene flow from shared ancestral polymorphism, and 3) linking introgression of figs and their chemical phenotypes to cases of pollinator host shifting. Our community-level approach is also essential to obtain the across-species replication necessary for robust statistical inference of diversification pattern and process across interacting fig and wasp taxa.
				</p>
			  </div>
           </div>
        </div>

        <div class="row">
            <div class="col-lg-12">
                <h3 class="page-header">Modeling Variation in Rates of Character Evolution
                </h3>
			  <div class="well">
				<p>Rates of Molecular Evolution — Accurate estimates of species divergence times are vital 
				to answering questions about historical biogeography, estimating diversification rates, and 
				identifying the causes of variation in rates of molecular evolution. However, obtaining reliable 
				divergence time estimates is confounded by the fact that the rate of evolution and time are intrinsically 
				linked. I am currently working on new models for estimating branch lengths in units proportional to time 
				without assuming a strict molecular clock. Our understanding of how substitution rates change over time 
				is still underdeveloped. My research in this field is aimed at identifying models that accommodate rate 
				change resulting from different processes. Such flexible models can provide more accurate estimates of 
				species divergence times. 
				I developed a model for estimating lineage-specific substitution rates that employs a Dirichlet 
				process prior (DPP) for modeling mixtures of data. Under this model, branches of a phylogenetic tree are 
				clustered into specific rate classes and the number of different rate classes and their associated rates 
				are treated as random variables. Thus, a strict molecular clock and a model where each lineage evolves 
				independently at a different rate are special cases of this model. Analyses of simulated data show that 
				divergence times and branch rates are more accurate under the DPP when compared with alternative models 
				for substitution rate variation. Furthermore, Bayesian inference under this model can identify how 
				phylogenetic lineages are partitioned according to substitution rate, information unavailable under 
				alternative priors. (See <a href="http://mbe.oxfordjournals.org/content/29/3/939">Heath et al., MBE 2012</a>)
				</p>
			  </div>
           </div>
        </div>


        <hr>
        <!-- Footer -->
        <footer>
            <div class="row">
                <div class="col-md-3 col-sm-12">
                    <a rel="license" href="https://github.com/phyloworks/phyloworks.github.io"><img style="border-width:0" src="images/Octocat.png" alt="" width="60" /></a><br>Hosted by <a href="https://github.com/">GitHub</a> Pages.
                	<br>Theme based on <a href="http://getbootstrap.com/">Bootstrap</a>, <a href="https://kuler.adobe.com/purple-haze-color-theme-57997/">Adobe Kuler 57997</a>, & <a href="http://paintstrap.com/">PaintStrap</a>.
                    <br>
                    <br>
                </div>
                <!-- /.col-lg-12 -->
                <div class="col-md-6 col-sm-12">
                	<center><a href="http://phylopic.org/image/6c9cb19d-1d8a-4215-88ba-d49cd4917a5e/"><img src="http://phylopic.org/assets/images/submissions/6c9cb19d-1d8a-4215-88ba-d49cd4917a5e.512.png" alt="" width="80" /></a>
                    <br>
                    <br>
                    </center>
                </div>
                <div class="col-md-3 col-sm-12">
                	<a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a><br />Except where otherwise noted, content on this site is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>.
                    <br>
                    <br>
                </div>
                <!-- /.col-lg-12 -->
            </div>
            <!-- /.row -->
        </footer>


    </div> <!-- /container -->


    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
    <script src="./js/bootstrap.min.js"></script>
    <script src="../../assets/js/docs.min.js"></script>
    <!-- IE10 viewport hack for Surface/desktop Windows 8 bug -->
    <script src="../../assets/js/ie10-viewport-bug-workaround.js"></script>
    <!-- Start of StatCounter Code for Default Guide -->
<script type="text/javascript">
var sc_project=10040953; 
var sc_invisible=1; 
var sc_security="62c272e1"; 
var scJsHost = (("https:" == document.location.protocol) ?
"https://secure." : "http://www.");
document.write("<sc"+"ript type='text/javascript' src='" +
scJsHost+
"statcounter.com/counter/counter.js'></"+"script>");
</script>
<noscript><div class="statcounter"><a title="website
statistics" href="http://statcounter.com/"
target="_blank"><img class="statcounter"
src="http://c.statcounter.com/10040953/0/62c272e1/1/"
alt="website statistics"></a></div></noscript>
<!-- End of StatCounter Code for Default Guide -->
  </body>
</html>
